crates/esthri/src/ops/sync.rs

```diff
@@ -162,7 +162,7 @@ pub async fn sync(
                 key
             );
 
-            sync_remote_to_local(s3, &bucket, &key, &path, &filters, opts).await?;
+            // sync_remote_to_local(s3, &bucket, &key, &path, &filters, opts).await?;
```

Comment:
we are having issues with stream delete.
i think some streaming functionalities removes the Send.

current non thread safe methods are sync, 
think `AsRef<Path>` is not Send safe,
`delete_streaming()` is causing a big problem

Reply:
fixed